src/UI/Analytics.elm

```diff
@@ -0,0 +1,58 @@
+module UI.Analytics exposing (Analytics(..), TableAnalytics(..), encode)
```

Comment:
Do we want to expose the type definition? I think this generates breaking changes if we add another branch in the type. so maybe it's better to move the TableAnalytics type to an internal type as far I can see, the external app does not need to know the type definition as the encode function is the one that transforms to a serializable data

Reply:
> Do we want to expose the type definition?

I really planned these types as public API because we may change how we encode them depending on the app or service.

This module is not supposed to know the integration (Mixpanel) or how we send data to it, so the encoder provided here is more like a suggestion than a rule.